src/XRP/serializer.ts

```diff
@@ -366,6 +376,34 @@ const serializer = {
     }
   },
 
+  /**
+   * Convert a {@link IssuedCurrencyAmount} to a JSON representation.
+   *
+   * @param issuedCurrencyAmount - The {@link IssuedCurrencyAmount} to convert.
+   * @returns A JSON representation of the input.
+   */
+  issuedCurrencyAmountToJSON(
+    issuedCurrencyAmount: IssuedCurrencyAmount,
+  ): IssuedCurrencyAmountJSON | undefined {
+    const currencyWrapper = issuedCurrencyAmount.getCurrency()
+    const value = issuedCurrencyAmount.getValue()
+    const issuer = issuedCurrencyAmount.getIssuer()?.getAddress()
+    if (currencyWrapper === undefined || value === '' || issuer === undefined) {
```

Comment:
are you sure that a missing `value` entry in an `IssuedCurrencyAmount` manifests as an empty string? Might be worth one more test that populates that the `currency` and `issuer` fields that hopefully still returns undefined, but if you're sure about it the logic seems obvious enough to me.

Reply:
Sure. Done.